src/Headers.php

```diff
@@ -283,7 +283,9 @@ public function get($name)
             return false;
         }
 
-        $class = ($this->getPluginClassLoader()->load(str_replace('-', '', $key))) ?: 'Laminas\Http\Header\GenericHeader';
+        $class = (
+            $this->getPluginClassLoader()->load(str_replace('-', '', $key))
+        ) ?: 'Laminas\Http\Header\GenericHeader';
```

Comment:
Brackets around expression are redundant, so I'd suggest the following formatting:
```php
 $class = $this->getPluginClassLoader()->load(str_replace('-', '', $key))
    ?: 'Laminas\Http\Header\GenericHeader';
```

Reply:
They are. They increase readability however, in my opinion